environment/src/test/java/com/sequenceiq/environment/environment/dto/UpdateAwsDiskEncryptionParametersDtoTest.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.environment.environment.dto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.environment.parameter.dto.AwsDiskEncryptionParametersDto;
+
+public class UpdateAwsDiskEncryptionParametersDtoTest {
+
+    @Test
+    void builderTest() {
```

Comment:
This does not seem to test `UpdateAwsDiskEncryptionParametersDto.builder()`.

Reply:
Addressed

Reply:
Addresssed.